### DISCUSSION TITLE
打包生成HTML能放到根目录下吗

### DISCUSSION THREAD

**Discussion by aasmn** [web-infra-dev/modern.js#6322] [Q&A] [answered]
如果配置能让打包生成的html直接放到 dist下，比如 dist/index.html

**Comment by kyrylobashtenko** [comment, marked as answer]
@aasmn https://modernjs.dev/en/configure/app/html/disable-html-folder.html
```
 html: {
    disableHtmlFolder: true,
  },
  output: {
    distPath: {
      html: '/',
    },
  },
  ```
I think at best you will get something like 
`dist/main.html`
cause it would build it per entry and since we might have multiple we need a different names for them.

**Reply by zllkjc** [reply]
For addition, if you want use `index.html` instead of `main.html`, you can use https://modernjs.dev/configure/app/source/mainentryname
